python_modules/jupedsim/jupedsim/models/social_force.py

```diff
@@ -0,0 +1,148 @@
+from dataclasses import dataclass
+
+import jupedsim.native as py_jps
+
+
+@dataclass(kw_only=True)
+class SocialForceModel:
+    """Parameters for Social Force Model
+
+    All attributes are initialized with reasonably good defaults.
+
+
+    Attributes:
+        bodyForce: describes the strength with which an agent is influenced by pushing forces from obstacles and neighbors in its direct proximity.
+        friction: describes the strength with which an agent is influenced by frictional forces from obstacles and neighbors in its direct proximity.
+    """
+
+    bodyForce: float = 120000
+    friction: float = 240000
```

Comment:
Is there a scale/unit to these parameters? If yes, it would be helpful to mention these, at least I have no clue what these values mean.

Reply:
there is now a comment containing the unit, a reference to the paper where the values are from and in some cases additional notes